setup.py

```diff
@@ -36,7 +36,7 @@
     "uwsgitop>=0.11",
     "uwsgi-tools>=1.1.1",
     "fuzzywuzzy>=0.18.0",
-    "python-ldap>=3.2.0,<3.3.0",
+    "python-ldap>=3.2.0,<3.5.0",
```

Comment:
https://github.com/CERNDocumentServer/cds-ils/pull/678 dependabot suggests to bump the minimal requirement